data/api/tmdb/src/main/java/io/filmtime/data/api/tmdb/VideoThumbnailExt.kt

```diff
@@ -43,13 +47,19 @@ fun TmdbShowDetailsResponse.toVideoDetail() =
     spokenLanguages = spokenLanguages?.map { it.englishName ?: "" }?.filter { it.isNotEmpty() }
       ?: listOf(),
     description = overview ?: "",
+    runtime = "",
+    releaseDate = "",
   )
 
 fun TmdbVideoResultResponse.toVideoThumbnail() = VideoThumbnail(
   ids = VideoId(traktId = null, tmdbId = id),
   title = title.orEmpty(),
   posterUrl = TMDB_BASE_IMAGE_URL.plus(posterPath),
-  year = releaseDate?.take(4)?.toInt() ?: 0,
+  year = if (releaseDate?.isNotEmpty() == true) {
+    releaseDate!!.take(4).toInt()
+  } else {
+    0
+  },
```

Comment:
I find this syntax more elegant compared to using `!!` 😅

```suggestion
  year = releaseDate?.takeIf { it.isNotEmpty() }?.take(4)?.toInt() ?: 0,
```